src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,10 @@
+import domain.PlusStrategyFactory;
+import domain.PlusStrategy;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String expressionStr) {
```

Comment:
이미 타입을 통해 String 변수라는 것을 유추해볼 수 있기에 변수명에는 타입을 제거해주셔도 좋을 것 같아요 😄 
```suggestion
    public static int splitAndSum(String expression) {
```

Reply:
타입으로 이미 유추가능하니 그렇겠네요 ㅎㅎ 하나배워갑니다